code/web/src/modules/style/Survey.js

```diff
@@ -37,6 +38,8 @@ class Survey extends Component {
           {question.images.map((option, j) => (
             <img 
               style={this.outlineSelection(option.value, question.answer)}
+              width={'200em'}
+              height={'300em'}
```

Comment:
Did you notice if having the height and width changed the aspect ratio of the images?

Reply:
Yeah, some of the images don't have the best aspect ratio and are looking a bit squished. Removing the height fixes this problem, but then the images aren't the same sizes. It will be good to fix this if we have enough time.